### PR TITLE
Xcode 10

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: Sources/Components/Broadcast/BroadcastItem.swift [hunk 1]
```diff
@@ -139,3 +139,9 @@ extension BroadcastItem: UITextViewDelegate {
         return false
     }
 }
+
+// Helper function inserted by Swift 4.2 migrator.
+fileprivate func convertToOptionalNSAttributedStringKeyDictionary(_ input: [String: Any]?) -> [NSAttributedString.Key: Any]? {
```
Comment (vadymmarkov) [inline Sources/Components/Broadcast/BroadcastItem.swift]: This function looks weird 😅 Should we perhaps change `BroadcastItem.Style.linkTextAttributes` to `[NSAttributedStringKey.foregroundColor: UIColor.primaryBlue]` (without rawValue)?

File: Sources/Fullscreen/EmptyView/EmptyView.swift [hunk 2]
```diff
@@ -225,7 +225,7 @@ public class EmptyView: UIView {
 
         let location = sender.location(in: self)
         let touchLocation = sender.location(in: objectView)
-        let touchOffset = UIOffsetMake(touchLocation.x - objectView.bounds.midX, touchLocation.y - objectView.bounds.midY)
+        let touchOffset = UIOffset.init(horizontal: touchLocation.x - objectView.bounds.midX, vertical: touchLocation.y - objectView.bounds.midY)
```
Comment (vadymmarkov) [inline Sources/Fullscreen/EmptyView/EmptyView.swift]: We can drop `init` here.

File: Sources/Components/Broadcast/BroadcastItem.swift (hunk 1 above)
Comment (3lvis) [reply Sources/Components/Broadcast/BroadcastItem.swift]: wow, Xcode!

**Comment by 3lvis** [comment]
Snapshot tests not working.

**Comment by Torsph** [comment]
@3lvis It became green for me if I did one run with `recordMode = true`.

**Comment by 3lvis** [comment]
@finn-no/ios This is ready for review. :D

**Comment by 3lvis** [comment]
Tests run normally on my computer.

**Comment by 3lvis** [comment]
We're having some issues with Broadcast snapshot testing because since it's using a Set the order keeps changing when the tests are done. I'm disabling snapshot testing for that component and will submit a PR to move from Set to Array in a different one/

**Pull request merged**